hazyCalculator.js

```diff
@@ -1,18 +1,31 @@
 function isSkippedValue(value) {
+  if (value === undefined || value === '') {
+    return true
+  }
+  const operators = ['+', '-', '*', '/']
+  if (operators.includes(value)) {
+    return false
+  }
   return !value
 }
 
 function isNumericValue(value) {
+  if (value === null) {
```

Comment:
Remember `isNan` will typecast to a number first and only give us false if that fails. This function works for all values except `''` (empty string) because it will typecast to `0`. So we just needed an explicit check for that by adding `&& value !== ''`